Fitness/Controllers/HomeController.swift

```diff
@@ -115,29 +121,54 @@ class HomeController: UIViewController {
             }
         }
     }
+    
+    func getCollectionViewCellSize(_ collectionView: UICollectionView, identifier: String) -> CGSize {
+        let allGymsCellSize = CGSize(width: (collectionView.bounds.width - 32.0 - 12.0)/2.0, height: 60.0)
+        let classCellSize = CGSize(width: 228.0, height: 195.0)
+        let discoverProsCellSize = CGSize(width: 280, height: 110.0)
+        let lookingForCellSize = CGSize(width: (collectionView.bounds.width-48)/2, height: ((collectionView.bounds.width-48)/2)*0.78)
+        let prosCellSize = CGSize(width: view.bounds.width, height: 145)
+        let todaysClassesCellSize = CGSize(width: collectionView.bounds.width, height: 227.0)
+        
+        if identifier == "classCell" {
+            return classCellSize
+        } else if identifier == "discoverProsCell" {
+            return discoverProsCellSize
+        } else if identifier == "allGymsCell" {
+            return allGymsCellSize
+        } else if identifier == "todaysClassesCell" {
+            return todaysClassesCellSize
+        } else if identifier == "lookingForCell" {
+            return lookingForCellSize
+        } else if identifier == "prosCell" {
+            return prosCellSize
+        }
+        return CGSize(width: 0, height: 0)
+    }
 }
 
 // MARK: CollectionViewDataSource
 extension HomeController: UICollectionViewDataSource {
 
     func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
-        if collectionView != mainCollectionView {
+        if collectionView.accessibilityIdentifier == Identifiers.todaysClassesCell {
             return gymClassInstances.count
+        } else if collectionView.accessibilityIdentifier == Identifiers.discoverProsCell {
```

Comment:
same comment as last PR, can we move these into the file that actually contains the collectionView?